build/build.sh

```diff
@@ -43,3 +44,20 @@ wkhtmltopdf \
   --quiet \
   output/index.html \
   output/manuscript.pdf
+
+echo "Exporting Word Docx manuscript"
```

Comment:
>  I'm not sure whether to tradeoff storage for ease of export until we know the target audience better.

What about an environment variable `BUILD_DOCX`? Then `if [ $BUILD_DOCX = "true" ]`, we build the DOCX. Therefore users can activate the feature by running

```
BUILD_DOCX=true sh build/build.sh
```

Or setting `BUILD_DOCX=true` via their Travis configuration or options.